src/script/components/calling/useCallAlertState.ts

```diff
@@ -24,11 +24,19 @@ type CallAlertState = {
   isGroupCall: boolean;
   showStartedCallAlert: (isGroupCall: boolean, isVideoCall?: boolean) => void;
   clearShowAlert: () => void;
+  qualityFeedBackModalShown: boolean;
+  setQualityFeedbackModalShown: (isVisible: boolean) => void;
```

Comment:
I would suggest a better name if you like:
`qualityFeedbackModalVisibility` and `setQualityFeedbackModalVisibility`

Reply:
A Boolean for me should begin with isXxx, hasXxx or something like that.